README.md

```diff
@@ -1,28 +1,158 @@
-# SubQuery - Dictionary 
+# 🚚 Ternoa Dictionary 
 
-This special SubQuery Project provides a dictionary of data that pre-indexes events on chain to dramatically improve indexing the performance of your own SubQuery Project, sometimes up to 10x faster.
+Ternoa Dictionary records all the native substrate on-chain data of the Ternoa blockchain: blocks, extrinsics, and events. It is a glossary of data that pre-indexes chain events, drastically improving the overall indexing performance.
 
-It scans over the network, and simply records the module and method for every event/extrinsic on each block - please see the standard entities in `schema.graphql`.
+You can start by familiarizing yourself with [Ternoa Explorer](https://explorer.ternoa.com/) as it was built using the Ternoa Dictionary.
 
-**If you want to create your SubQuery Dictionary to speed up indexing of your own Substrate chain, fork this project and let us know**
+Don't forget to have fun with it, Cheers 🍻
 
-# Geting Started
-### 1. Install dependencies
-```shell
-yarn
+**Table of Contents :** 
+
+- [Introduction](#🤔-introduction)
+  - [Dictionary](#dictionary)
+  - [Error Reporting](#error-reporting)
+- [Installation](#installation)
```

Comment:
They are some typo issues in the subsections (e.g. '#' is missing for the 2nd subsection, `Select Testnet Version` is not the correct title).

IMO you can remove all of those subsections; `installation` is enough

Reply:
DOne